Holovibes/includes/api/information_api.hh

```diff
@@ -4,135 +4,145 @@
-                                        "Edgar Delaporte",
-
-                                        "Adrien Langou",
-                                        "Julien Nicolle",
-                                        "Sacha Bellier",
-                                        "David Chemaly",
-                                        "Damien Didier",
-
-                                        "Philippe Bernet",
-                                        "Eliott Bouhana",
-                                        "Fabien Colmagro",
-                                        "Marius Dubosc",
-                                        "Guillaume Poisson",
-
-                                        "Anthony Strazzella",
-                                        "Ilan Guenet",
-                                        "Nicolas Blin",
-                                        "Quentin Kaci",
-                                        "Theo Lepage",
-
-                                        "Loïc Bellonnet-Mottet",
-                                        "Antoine Martin",
-                                        "François Te",
-
-                                        "Ellena Davoine",
-                                        "Clement Fang",
-                                        "Danae Marmai",
-                                        "Hugo Verjus",
-
-                                        "Eloi Charpentier",
-                                        "Julien Gautier",
-                                        "Florian Lapeyre",
-
-                                        "Thomas Jarrossay",
-                                        "Alexandre Bartz",
-
-                                        "Cyril Cetre",
-                                        "Clement Ledant",
-
-                                        "Eric Delanghe",
-                                        "Arnaud Gaillard",
-                                        "Geoffrey Le Gourrierec",
-
-                                        "Jeffrey Bencteux",
-                                        "Thomas Kostas",
-                                        "Pierre Pagnoux",
-
-                                        "Antoine Dillée",
-                                        "Romain Cancillière",
-
-                                        "Michael Atlan"};
-
-/*! \brief Gets the credits
- *
- * \return const std::vector<std::string> credits in columns
- */
-constexpr std::vector<std::string> get_credits()
+class InformationApi : public IApi
 {
-    std::vector<std::string> res{"", "", ""};
 
-    size_t nb_columns = 3;
-    for (size_t i = 0; i < authors.size(); i++)
-        res[i % nb_columns] += authors[i] + "<br>";
+  public:
+    InformationApi(const Api* api)
+        : IApi(api)
+    {
+    }
 
-    return res;
-}
+#pragma region Credits
+
+    /*! \brief Authors of the project */
+    static inline std::vector<std::string> authors{"Titouan Gragnic",
```

Comment:
Idk but shouldn't this be private as well ? (accessible by const getter or just hidden)